src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -547,16 +555,21 @@ private void finishImportList() {
         String currentGroup = getImportGroup(firstImport.isStaticImport(),
                 firstImport.getImportFullPath());
         int currentGroupNumber = customImportOrderRules.indexOf(currentGroup);
+        ImportDetails previousImportObjectFromCurrentGroup = null;
         String previousImportFromCurrentGroup = null;
 
-        for (ImportDetails importObject : importToGroupList) {
+        for (final ImportDetails importObject : importToGroupList) {
```

Comment:
Please remove `final` from for loop variable.

Reply:
Done!